pkg/k8s/pipelineRun.go

```diff
@@ -43,15 +44,26 @@ type pipelineRun struct {
 
 // NewPipelineRun creates a managed pipeline run object
 // the provided PipelineRun object is never modified and copied as late as possible
```

Comment:
The doc is not sufficient anymore. It does not describe the role of the factory and its absence.

Reply:
Doc updated